openshift/knative-eventing-encryption-auth.yaml

```diff
@@ -0,0 +1,5 @@
+spec:
```

Comment:
should we name the file `-patch` I was looking where this partial file was used, and saw the patch. I like the idea, but the file name did not directly hint this